src/go/package/app/models/tag_test.go

```diff
@@ -0,0 +1,54 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/goravel/framework/database/orm"
+	"github.com/goravel/framework/support/carbon"
+	"github.com/stretchr/testify/suite"
+
+	protopackage "market.goravel.dev/proto/package"
+)
+
+type TagSuite struct {
+	suite.Suite
+	tag *Tag
+}
+
+func TestTagSuite(t *testing.T) {
+	suite.Run(t, new(TagSuite))
+}
+
+func (s *TagSuite) SetupTest() {
+	s.tag = NewTag()
+}
+
+func (s *TagSuite) TestToProto() {
```

Comment:
Good habits, but the function is very simple, don't need to test it.

Reply:
removed it